lib/tizen_tpk.dart

```diff
@@ -144,41 +147,43 @@ class TizenManifest {
       _manifest.children.insert(0, builder.buildFragment());
     }
     return _manifest.findElements('profile').first;
-  }
+  }();
 
-  /// The profile name representing the device type.
+  /// The profile name representing the target device type.
   String get profile => _profile.getAttribute('name')!;
   set profile(String value) => _profile.setAttribute('name', value);
 
-  String? _applicationId;
-
-  /// The unique application ID used for launching and terminating applications.
-  String get applicationId {
-    if (_applicationId == null) {
-      final Iterable<XmlElement> applications = _manifest.children
-          .whereType<XmlElement>()
-          .where((XmlElement element) =>
-              element.name.local.endsWith('-application') &&
-              element.getAttribute('appid') != null);
-      if (applications.isEmpty) {
-        throwToolExit('Found no *-application element with appid attribute in '
-            'tizen-manifest.xml.');
-      }
-      final XmlElement application = applications.first;
-      final String tag = application.name.local;
-      if (tag != 'ui-application' && tag != 'service-application') {
-        globals.printTrace(
-            'tizen-manifest.xml: <$tag> is not officially supported.');
-      }
-      _applicationId = application.getAttribute('appid');
-      if (applications.length > 1) {
-        globals.printTrace(
-            'tizen-manifest.xml: Found ${applications.length} application declarations. '
-            'Using the first one: <$tag appid="$_applicationId">');
-      }
+  late final Iterable<XmlElement> _applications = () {
```

Comment:
If you're going to use only the first element of applications anyway, why do you have to take the list of applications?
Are you considering supporting the multi-package?

Reply:
As you might know we already support [multi-app](https://github.com/flutter-tizen/flutter-tizen/issues/143) packages.

As you said it's perfectly fine to keep only the first element of the list because the other elements are not used at the moment, but I was not sure how to name the variable as a singular noun. (Maybe `_primaryApplication`?)

Reply:
If there is a possibility that the applications list will be used later, it may be possible to keep it. but  if you change it `_primaryApplication` is good to me.

Reply:
It might be used in the future, e.g. to figure out if the package is a multi-app package.